test/selenium/configurations.rb

```diff
@@ -36,11 +36,11 @@ def common_configuration_for_firefox_extension
   Capybara.app_host = "chrome://privly"
   @@privly_applications_folder_path = Capybara.app_host + "/content/privly-applications/"
   puts "Packaging the Firefox Extension"
-  system( "cd ../../../../../ && pwd && ./package.sh && cd chrome/content/privly-applications/test/selenium" )
+  system("cd ../../../../../ && pwd && jpm xpi && cd chrome/content/privly-applications/test/selenium")
 
   # Load the Firefox driver with the extension installed
   @profile = Selenium::WebDriver::Firefox::Profile.new
-  @profile.add_extension("../../../../../PrivlyFirefoxExtension.xpi")
+  @profile.add_extension("../../../../../@privly-0.5.0.xpi")
```

Comment:
Nope, but I can do something like [this](https://github.com/privly/privly-applications/blob/master/test/selenium/run_all.rb#L58) to find out the version number and then use the correct file name.